### DISCUSSION TITLE
Change delay for returning to page Home

### DISCUSSION THREAD

**Discussion by Njamotep** [Blackymas/NSPanel_HA_Blueprint#575] [General]
Hi! Great automation, really fantastic work! 

I'm trying to find out if it is possible to change the ~8000ms or so returning to page Home? I feel it is somewhat short as is, especially for QR code when fumbling with a phone. Tried to look through your absolutely MASSIVE code but didn't find what I was looking for.

Thanks

**Comment by lloydbayley** [comment]
Hi! This was recently discussed in issue #555 . Have a look in there and hopefully that might answer your question! :)

**Comment by deejaybeam** [comment]
just set the page-timeout on your device

**Reply by Njamotep** [reply]
Thanks! Can't believe I missed it. Been looking everywhere, except there.. oh well.
